en/docs/updates/customized-dockerfile.md

```diff
@@ -0,0 +1,57 @@
+### Creating Custom Docker images for Updated Products
+The `create-docker` command builds a Docker image of an updated product pack using the Docker resources developed by WSO2. However, the following steps should be followed if any customizations are required.
+
+In this example, we will assume that we are creating a custom Docker image for an updated product of WSO2 API Manager 3.0.0<br>
+    1. Go to the product Docker resources repository on GitHub (e.g., docker-apim).<br>
```

Comment:
Shall we add a hyperlink to the example? "(e.g. [docker-apim](https://github.com/wso2/docker-apim/))"